netutils/ip.py

```diff
@@ -387,3 +387,23 @@ def get_usable_range(ip_network):
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def ip_to_ptr(ip_address):
+    """Generates a ptr record from a given IP address.
+
+    Args:
+        ip4_address (str): IPv4 address.
```

Comment:
Can we update this?

Reply:
I should've caught that. Thanks Jeff. Updated in case we do use this PR.